### PR TITLE
fix: correct operation id

### PR DESCRIPTION
**Pull request by BCerki**
https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=51077093

There's a lot of work in flight re the operation form right now, so this PR just fixes the bug. I'll make a tech debt card to finish moving data transformation out of the front-end and into the schema.

### CODE REVIEW

File: bc_obps/registration/schema/operation.py [hunk 1]
```diff
@@ -73,11 +73,28 @@ class OperationOut(ModelSchema):
     bcghg_id: Optional[str] = None
     opt_in: Optional[bool] = None
     verified_at: Optional[date] = None
-    point_of_contact: Optional[ContactSchema]
+    # point of contact handling
+    first_name: Optional[str] = Field(None, alias="point_of_contact.first_name")
+    last_name: Optional[str] = Field(None, alias="point_of_contact.last_name")
+    email: Optional[str] = Field(None, alias="point_of_contact.email")
+    # phone_number: Optional[str] = Field(None, alias="point_of_contact.phone_number")
```
Comment (BCerki) [inline bc_obps/registration/schema/operation.py:80]: b need to fix

**Review by Sepehr-Sobhani — APPROVED** [review]
Great stuff! 🥳
As a later refactor, I recommend retaining the `point_of_contact: Optional[ContactSchema] = None` attribute and eliminating the resolver (as it is managed within the `ContactSchema`). This approach allows us to manipulate the `point_of_contact object` directly on the frontend, contributing to a cleaner code structure, in my opinion.

**Comment by BCerki** [comment]
> Great stuff! 🥳 As a later refactor, I recommend retaining the `point_of_contact: Optional[ContactSchema] = None` attribute and eliminating the resolver (as it is managed within the `ContactSchema`). This approach allows us to manipulate the `point_of_contact object` directly on the frontend, contributing to a cleaner code structure, in my opinion.

Sorry, I don't think I follow! Incorrectly manipulating the point_of_contact object on the frontend was what caused this bug. I was going to make a tech debt card to move all manipulation into the schema instead of the frontend (here I just did the one part), but if there's a better way, I'll make a card for that instead.

**Comment by Sepehr-Sobhani** [comment]
> > Great stuff! 🥳 As a later refactor, I recommend retaining the `point_of_contact: Optional[ContactSchema] = None` attribute and eliminating the resolver (as it is managed within the `ContactSchema`). This approach allows us to manipulate the `point_of_contact object` directly on the frontend, contributing to a cleaner code structure, in my opinion.
> 
> Sorry, I don't think I follow! Incorrectly manipulating the point_of_contact object on the frontend was what caused this bug. I was going to make a tech debt card to move all manipulation into the schema instead of the frontend (here I just did the one part), but if there's a better way, I'll make a card for that instead.

Apologies for the confusion. I'm game to move the data transformation to the schema or backend. Please let me know when you plan to start working on it, so we can brainstorm together.

**Comment by BCerki** [comment]
> > > Great stuff! 🥳 As a later refactor, I recommend retaining the `point_of_contact: Optional[ContactSchema] = None` attribute and eliminating the resolver (as it is managed within the `ContactSchema`). This approach allows us to manipulate the `point_of_contact object` directly on the frontend, contributing to a cleaner code structure, in my opinion.
> > 
> > 
> > Sorry, I don't think I follow! Incorrectly manipulating the point_of_contact object on the frontend was what caused this bug. I was going to make a tech debt card to move all manipulation into the schema instead of the frontend (here I just did the one part), but if there's a better way, I'll make a card for that instead.
> 
> Apologies for the confusion. I'm game to move the data transformation to the schema or backend. Please let me know when you plan to start working on it, so we can brainstorm together.

I made a tech debt card, probably no one will get to it until post-MVP at least: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=51484521

**Pull request merged**